internal/controller/generic_tunnel_reconciler.go

```diff
@@ -19,7 +18,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
-const CREDENTIALS_JSON_FILENAME string = "credentials.json"
+const (
+	CREDENTIALS_JSON_FILENAME string = "credentials.json"
+	CLOUDFLARED_LATEST_IMAGE  string = "cloudflare/cloudflared:latest"
```

Comment:
Let's name constants in camel case, https://stackoverflow.com/questions/22688906/go-naming-conventions-for-const

Reply:
Revive linter also expect camel case